selfdrive/car/subaru/carcontroller.py

```diff
@@ -56,7 +56,7 @@ def update(self, CC, CS, now_nanos):
       cruise_brake = clip(apply_brake, CarControllerParams.BRAKE_MIN, CarControllerParams.BRAKE_MAX)
     else:
       cruise_throttle = CarControllerParams.THROTTLE_INACTIVE
-      cruise_rpm = CarControllerParams.RPM_INACTIVE
+      cruise_rpm = CarControllerParams.RPM_MIN
```

Comment:
I wouldn't call the other value `RPM_INACTIVE`, as that matches the safety value. GM calls it `ZERO_GAS`

Reply:
I'll merge the fix and then standardize the names in another PR